src/library-aux/scala/Any.scala

```diff
@@ -27,7 +27,7 @@ package scala
  *     w.print()
  * }}}
  *
- * See the [[http://docs.scala-lang.org/sips/completed/value-classes.html value classes guide]] for more
+ * See the [[http://docs.scala-lang.org/sips/completed/value-classes.html SIP-15: Value Classes]] for more
```

Comment:
better to link to http://docs.scala-lang.org/overviews/core/value-classes.html here, actually, IMO